editors/sc-ide/core/settings/theme.cpp

```diff
@@ -214,8 +214,14 @@ void Theme::setFormat(const QString & key, const QTextCharFormat & newFormat)
                     QColor(Qt::transparent): newFormat.background().color();
 
     if (i == mFormats.end()) {
-        qDebug() << __func__ << "Failed to find key " << key;
-        return;
+        qDebug() << 
+#ifndef WIN32
```

Comment:
WIN32 is not a standard, use _WIN32 instead
http://sourceforge.net/p/predef/wiki/OperatingSystems/